core/templates/services/UpgradedServices.ts

```diff
@@ -66,6 +66,8 @@ import { CollectionCreationBackendService } from
 import { ComputeGraphService } from 'services/compute-graph.service';
 import { ConceptCardObjectFactory } from
   'domain/skill/ConceptCardObjectFactory';
+import { ConceptCardBackendApiService } from
```

Comment:
Please sort the imports -- this should be placed above ConceptCardObjectFactory

Reply:
done